example/accel/CMakeLists.txt

```diff
@@ -17,7 +17,9 @@ if(NOT CELERITAS_USE_Geant4)
     "to be enabled in Celeritas")
 endif()
 
-include(CudaRdcUtils)
+# Use Celeritas CUDA RDC macros to correctly link libraries using VecGeom
+# NOTE: until vecgeom/g4vg get updated, we must use the Celeritas load path
+include("${Celeritas_DIR}/CudaRdcUtils.cmake")
```

Comment:
humm .. that should not be need :(

Reply:
i.e. With the PR we recently merged, it should have installed the Celeritas version and it should have been preferred ...

Reply:
> VecGeom if you let Celeritas loaded it.

We could/should fix that.

Reply:
Yes, I think so.

Reply:
I guess we should've been doing this all along 🤔